sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+		<IsExtensionClientLibrary>true</IsExtensionClientLibrary>
+		<Version>1.0.0-beta.1</Version>
+		<Description>Triggers for Azure AD Authentication event custom extensions. Lets you focus on your business logic.</Description>
+		<GeneratePackageOnBuild>False</GeneratePackageOnBuild>
```

Comment:
Should remove

Reply:
Thanks @JoshLove-msft are you referring to just 'GeneratePackageOnBuild'

Reply:
Yes

Reply:
Done